docs/commandline.rst

```diff
@@ -4,3 +4,37 @@
 
 .. autoprogram:: doctr.__main__:get_parser()
    :prog: doctr
+
+
+Configuration
+-------------
+
+In addition to command line arguments you can configure ``doctr`` using the
+``.travis.yml`` files. Command line arguments take precedence over the value
+present in the configuration file.
+
+The configuration parameters available from the ``.travis.yml`` file often
+mirror their command line siblings except dashes ``-`` are replaced by
+underscores ``_``. Some parameters are not available from configuration files
+for security reasons.
```

Comment:
All the parameters are paths (strings). Unexpected stuff can happen in YAML if you don't quote a string and it looks like another literal (like a float), but that's unlikely to happen here. I can't think of any other issues.